spec/support/factory_bot.rb

```diff
@@ -0,0 +1,3 @@
+RSpec.configure do |config|
```

Comment:
No need additional file. Configure this on spec_helper file

Reply:
fixed it